test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -262,6 +266,15 @@ export default class BaseFormObject {
   }
 
   async selectValueChozenDropdown(fieldName, index) {
+    if (userVariables.v3) {
+      const selectEle = await this.el.find(`[data-se="${fieldName}"]`);
+
+      await this.t.click(selectEle);
+      
+      const option = selectEle.child().nth(index);
+      await this.t.click(option);
+      return;
+    }
```

Comment:
Outside of using `fireEvent` I am not sure how to trigger the change for a dropdown / radio, so I went with this but looking for feedback.